src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -21,6 +29,54 @@ type Props = {
 export class MenuButtonsMetaInfo extends React.PureComponent<Props> {
   render() {
     const meta = this.props.profile.meta;
+    const overheads: ?(ProfilerOverhead[]) = this.props.profile
+      .profilerOverhead;
+    const calculatedStats: Map<string, ProfilerStats> = new Map();
+    // These 3 values only have single values, so we are not using ProfilerStats class for them.
+    let overheadDurations = 0;
+    let overheadPercentage = 0;
+    let profiledDuration = 0;
+    let totalSamplingCount = 0;
+
+    // Older profiles(Before FF 70) don't have any overhead info. Don't show anything if
+    // that's the case.
+    if (overheads) {
+      // Overhead keys that have min/max/mean values to loop.
+      const statKeys = [
+        'Overhead',
+        'Cleaning',
+        'Counter',
+        'Interval',
+        'Lockings',
+      ];
+
+      for (const overhead of overheads) {
+        const { statistics } = overhead;
+        const { samplingCount } = statistics;
+
+        // Calculation the single values without any loop, it's not worth it.
+        overheadDurations += statistics.overheadDurations * samplingCount;
+        overheadPercentage += statistics.overheadPercentage * samplingCount;
+        profiledDuration += statistics.profiledDuration * samplingCount;
+        totalSamplingCount += samplingCount;
```

Comment:
I didn't like this but we had to calculate these individual fields separately since they don't have min/max/mean values. I used `ProfilerStats` helper class to calculate the others.